google-cloud-clients/google-cloud-storage/src/main/java/com/google/cloud/storage/BucketInfo.java

```diff
@@ -848,6 +849,8 @@ void populateCondition(Rule.Condition condition) {
 
     abstract Builder setMetageneration(Long metageneration);
 
+    abstract Builder setLocationType(String location);
```

Comment:
This should also be `setLocationType(String locationType)`

Reply:
done